plugins/alerts/frontend/public/stylesheets/vue-main.scss

```diff
@@ -0,0 +1,56 @@
+#alerts-view {
+    .text-heading.title {
```

Comment:
we have a class called - **text-smallish** that gives 13px font size, so maybe you can use that.